test/integration/util/util.go

```diff
@@ -107,14 +106,12 @@ func StartScheduler(clientSet clientset.Interface, kubeConfig *restclient.Config
 
 // StartFakePVController is a simplified pv controller logic that sets PVC VolumeName and annotation for each PV binding.
 // TODO(mborsz): Use a real PV controller here.
-func StartFakePVController(clientSet clientset.Interface) ShutdownFunc {
-	ctx, cancel := context.WithCancel(context.Background())
-
+func StartFakePVController(ctx context.Context, clientSet clientset.Interface) ShutdownFunc {
+	ctx, cancel := context.WithCancel(ctx)
```

Comment:
Both `StartScheduler` and `StartFakePVController` accept a `ctx` now and both set up cancelation. I suspect that this duplication can be avoided by moving the `context.WithCancel(ctx)` into the caller and then removing `WithCancel` and the `ShutdownFunc` return value from these functions.

Reply:
This is for intended, hope to tear down the kube components almost the same time.

Reply:
So why not use the same cancellation context for both?

Reply:
The function signatures also become simpler.

Reply:
`Make the function signatures simpler` makes sense to me. Update as suggested.